PL2/PL2-shared/src/test/java/nl/tudelft/pl2016gr2/model/OriginalGraphTest.java

```diff
@@ -0,0 +1,49 @@
+package nl.tudelft.pl2016gr2.model;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+import java.util.ArrayList;
+
+public class OriginalGraphTest extends GraphTest {
+  
+  private OriginalGraph curGra;
+
+  @Override
+  public GraphInterface getInstance() {
+    OriginalGraph gra = new OriginalGraph();
+    ArrayList<String> ar = new ArrayList<String>();
+    ar.add("aba");
+    Node n1 = new Node(2, 8, ar, 5);
+    Node n2 = new Node(5, 8, null, 9);
+    gra.addNode(n1);
+    gra.addNode(n2);
+    gra.setGenoms(ar);
+    this.curGra = gra;
+    return gra;
+  }
+  
+  @Test
+  public void testGetNodes() {
+    assertEquals(curGra.getNodes().size(), 2);
```

Comment:
The first parameter of assertEquals should be the expected value and the second parameter should be the actual value:
<code>assertEquals(2, curGra.getNodes().size());</code>